Clinical-Genomics/scout
# Szilvas' Cancer SVs

**This PR adds a functionality - Cancer SVs!**

Thank you @szilvajuhos! This is Good Stuff that we should try to get into the base repo. We should add some tests and unify the views a little bit before merging, but you've done a really Good Thing already!

How to test:

1. Load a manta cancer SV track
1. Note that a cancer sv button appears on the case
1. Look, click and filter through functionality on the variants and variant pages for the cancer_sv to see that all is working as far as can be expected from current annotation status.
1. Compare to other variants forms and note any glaring functionality differences

One immediate thing is the addition of a test file, ideally from the previously existing test case, but anything fit for the public domain is fine really. @moonso is this something we have / can produce with small effort? I'm guessing the current bams are not complete enough that I could just fire up manta on and get something meaningful? Otherwise I'm sure @hassanfa or @szilvajuhos have test cases that we could use.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] tests executed by CR, Travis CI
